src/styles/blocks/catalog.scss

```diff
@@ -12,14 +12,21 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
-    height: 300px;
-    border: 1px solid #ccc;
+    width: $width-size-card;
+    height: $height-size-card;
+    border: $frame;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $animation-speed;
```

Comment:
specify what exactly you'd like to change setting transition-property to avoid using default value `all` for transition-property